website/pages/_document.js

```diff
@@ -1,4 +1,4 @@
-import Document, { Head, Main, NextScript } from 'next/document'
+import Document, { Html, Head, Main, NextScript } from 'next/document'
```

Comment:
After upgrading `next` to `9.5.5.`, I started seeing a warning `Missing component: <Html />`. So I followed [NextJS's guide on fixing the error](https://github.com/vercel/next.js/blob/master/errors/missing-document-component.md).